pallets/elections/src/tests.rs

```diff
@@ -18,3 +31,44 @@ fn test_elect() {
         assert_eq!(elected.unwrap(), &[(1, supp.clone()), (2, supp)]);
     });
 }
+
+#[test]
+fn migration_from_v0_to_v1_works() {
+    new_test_ext(vec![1, 2, 3], vec![]).execute_with(|| {
```

Comment:
Can we add another test to make sure we discard previous values?

Reply:
I'm having this fundamental problem of not understanding how the test works. So for the test to make sense it would need to migrate from the old version of the pallet to the new one. How does it have access to the old one? The mock seems to have the new one?